src/main/kotlin/pl/edu/uj/ii/ksi/mordor/services/ExternalUserService.kt

```diff
@@ -0,0 +1,32 @@
+package pl.edu.uj.ii.ksi.mordor.services
+
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import pl.edu.uj.ii.ksi.mordor.persistence.entities.Role
+import pl.edu.uj.ii.ksi.mordor.persistence.entities.User
+import pl.edu.uj.ii.ksi.mordor.persistence.repositories.UserRepository
+
+@Service
+class ExternalUserService(private val userRepository: UserRepository) {
+    companion object {
+        private val logger = LoggerFactory.getLogger(LdapAttributesMapper::class.java)
```

Comment:
I suppose it should be `ExternalUserService::class.java`.

Reply:
fixed